examples/em-server-multi-tier-cronjob/docker-compose.yml

```diff
@@ -11,6 +11,7 @@ services:
       - DB_HOST=db
       - WEB_SERVER_HOST=0.0.0.0
       - SERVER_BRANCH=ceo_ebike_project
+      - STUDY_CONFIG=stage-program
```

Comment:
I was going to ding this because I believed that the STUDY_CONFIG in the NREL environment would be set to `stage`, not `stage-program`.
Note that there is one stage environment (https://openpath-stage.nrel.gov), we just support two different configs for it because it is a staging environment and we want to test it out.

But I looked at the actual logs from the viz_scripts, and it looks like it is actually set to `stage-study`

```
Running crontab as a AWS scheduled tasks, setting python path
About to download config from https://raw.githubusercontent.com/e-mission/nrel-openpath-deploy-configs/main/configs/stage-study.nrel-op.json
```